ClosedXML.Report.sln

```diff
@@ -16,40 +16,26 @@ EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
-		Release 3.5|Any CPU = Release 3.5|Any CPU
-		Release 4.5|Any CPU = Release 4.5|Any CPU
-		Release 4.6|Any CPU = Release 4.6|Any CPU
+		Release.Signed|Any CPU = Release.Signed|Any CPU
```

Comment:
There are many irrelevant changes displayed in this PR. Perhaps you should merge `ClosedXML:develop` into `b0bi79:develop` first.

Reply:
I already did it, I won't understand why so.

Reply:
You merged your `master` to your `develop`. You also need merge `upstream:develop` into `origin:develop` and then push it.
![image](https://user-images.githubusercontent.com/19576939/40113192-acf02e32-5919-11e8-83b5-8ae7c704b988.png)